app/src/main/java/com/example/trashminder/presentation/createdReminder/HomeFragment.kt

```diff
@@ -63,4 +87,72 @@ class HomeFragment : Fragment() {
             }
         }
     }
+
+    @RequiresApi(Build.VERSION_CODES.O)
+    @Composable
+    private fun CheckNotificationPermission(it: Reminder, context: Context) {
```

Comment:
Since this is a view that will be displayed to the user, I would expect not to see a verb in the name. The naming seems to be better suited for a function rather than a view (I know technically speaking a composable is a function, but I think you get my point)